### PR TITLE
chore: Add latest instance types that support EFA

### PR DESCRIPTION
**Pull request by bryantbiggs**
### Issue

- N/A

### Description of changes

- Add latest instance types that support EFA

### Checklist
- [ ] Added/modified documentation as required (such as the `README.md` for modified charts)
- [ ] Incremented the chart `version` in `Chart.yaml` for the modified chart(s)
- [ ] Manually tested. Describe what testing was done in the testing section below
- [ ] Make sure the title of the PR is a good description that can go into the release notes

### Testing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by Pavani-Panakanti — APPROVED** [review]

File: stable/aws-efa-k8s-device-plugin/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: aws-efa-k8s-device-plugin
 description: A Helm chart for EFA device plugin.
-version: v0.5.4
-appVersion: "v0.5.4"
+version: v0.5.5
+appVersion: "v0.5.5"
```
Comment (Pavani-Panakanti) [inline stable/aws-efa-k8s-device-plugin/Chart.yaml:5]: appVersion should stay at "v0.5.4" which is the latest image version ?

Comment (jaydeokar) [reply stable/aws-efa-k8s-device-plugin/Chart.yaml:5]: Yes, let's keep appVersion same as the efa-plugin version so that we can keep a track

Comment (bryantbiggs) [reply stable/aws-efa-k8s-device-plugin/Chart.yaml]: ah yes - makes sense. corrected in https://github.com/aws/eks-charts/pull/1151/commits/7f82abe15fd15068ccbf8724b0fa4a353965c4e4

**Review by jaydeokar — APPROVED** [review]
lgtm

**Pull request merged**